deploy/kiali/kiali_cr.yaml

```diff
@@ -159,6 +162,38 @@ spec:
 #      ldap_use_ssl: false
 #      ldap_user_filter: "(cn=%s)"
 #      ldap_user_id_key: "cn"
+#
+#    ---
```

Comment:
Move this marker so it appears immediately above the new section's settings. The idea is that the `---` marker appears just above the actual section settings so the user's eyes are drawn to the values and knows what has to be uncommented to set the values. See the comment in lines 16 thru 19 that explains this.

```suggestion
```

Reply:
OK, it's moved down.